### ISSUE TITLE
Copy & Paste does not work in GB Studio 3 on Ubuntu 20.04

### ISSUE THREAD

**Issue by bunnymnemonic** [chrismaltby/gb-studio#869]
Copy & Paste functions do not work in GB Studio 3 on Ubuntu 20.04.
You can reproduce the bug by clicking for example on "Copy Scene". "Paste Scene" does not appear at all.
Tried it in a new empty project, in a sample project, and in an existing old project as well
![Screenshot_20211213_114759](https://user-images.githubusercontent.com/23193196/145799054-5d5be9b6-60d2-474d-b144-e45920ef52d2.png)
.

**Comment by RichardULZ** [comment]
Please try to use the issue template to keep bug reports consistent.
Would you be able to test this on 2.0beta5 from code>releases? 
Is this a new issue or something on multiple versions.

**Comment by bunnymnemonic** [comment]
Sorry for the inconsistency. I edited the original post to fit the template.
Previous versions including 2.0beta5 did not have this issue, so this is a new problem. (I downgraded GB Studio to 2.0beta5, by the way.)

**Comment by DuendeInexistente** [comment]
+1ing the issue, I have it too in the same version

**Comment by RichardULZ** [comment]
I overheard talk about trying to use clipboard data so you didn't "accidentally" paste raw json elsewhere, maybe that could be part of the issue? linux doesn't support raw data clipboard?
it might have been to avoid pasting it in a text box.

I'm concerned if we can't plain text paste json, in say notepad, there are a lot of tools blocked out from editing data, like collision or mettasprites, or just seeing how data looks under the hood.

It's very common for apps like after effects or nuke to create and paste raw text data, sometimes with a note on header to say "after effects keyframe data" first.

**Comment by bunnymnemonic** [comment]
I don't think it is because of raw data clipboard. As I change back to version 2.0beta5, it immediately works, and my clipboard data has not been changed. Was there any change in clipboard management in versions 3.0.0 and 3.0.1?

**Comment by DuendeInexistente** [comment]
Maybe it was a change in node or WebKit? As of a week or two ago I can't
paste images directly into discord (I use the browser version) and have to
manually pick them in my clipboard manager, which seems to process them in
a way chrome likes .

**Comment by bunnymnemonic** [comment]
As I can see, clipboard management has been rewritten in 3.0.0 and 3.0.1. Copying might be the problem, because I can copy a scene from 2.0beta5, and "Paste" appears in 3.0.0. Of course, it cannot paste it because it is a different format (clipboard format and types has been introduced in the new version). I cannot fully understand the code but the problem might hide there.

**Comment by bunnymnemonic** [comment]
I have just tested GB Studio 3.0.1 under Ubuntu 21.10 and Kubuntu 21.10. Clipboard does not work in neither.
